src/axom/bump/TopologyMapper.hpp

```diff
@@ -405,6 +405,8 @@ AXOM_HOST_DEVICE double shapeOverlap(const VariableShape<T, 3> &shape1,
  * \tparam SrcMatsetView The view type for the source matset.
  * \tparam TargetTopologyView The view type for the target topology.
  * \tparam TargetCoordsetView The view type for the target coordset.
+ * \tparam MAX_VERTS The maximum number of vertices allowed in a polygon. This
+ *                   value is used only in 2D.
```

Comment:
Would it make sense to name it something like `MAX_VERTS_2D`?

Reply:
I made that change.